app/lib/geo.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# Shared geographic calculations
+module Geo
```

Comment:
Would prefer a more descriptive name than `Geo`.

Reply:
I used Geo because it is a container for shared geographic methods.  To me it is short and descriptive.  But open for suggestions.

Reply:
not addressed in latest commit.